Prism mock command fails with dependancy issue

## Describe the bug

Updated the prism npm package today and then tried using the mock command, the command failed with the following error:

```
internal/modules/cjs/loader.js:775
    throw err;
    ^

Error: Cannot find module 'lodash/fp'
Require stack:
- /usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/validator/validators/security/index.js
- /usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/validator/index.js
- /usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/mocker/callback/callbacks.js
- /usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/mocker/index.js
- /usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/forwarder/index.js
- /usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/index.js
- /usr/local/lib/node_modules/@stoplight/prism-cli/dist/util/createServer.js
- /usr/local/lib/node_modules/@stoplight/prism-cli/dist/commands/mock.js
- /usr/local/lib/node_modules/@stoplight/prism-cli/dist/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:772:15)
    at Function.Module._load (internal/modules/cjs/loader.js:677:27)
    at Module.require (internal/modules/cjs/loader.js:830:19)
    at require (internal/modules/cjs/helpers.js:68:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/validator/validators/security/index.js:7:14)
    at Module._compile (internal/modules/cjs/loader.js:936:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:947:10)
    at Module.load (internal/modules/cjs/loader.js:790:32)
    at Function.Module._load (internal/modules/cjs/loader.js:703:12)
    at Module.require (internal/modules/cjs/loader.js:830:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/validator/validators/security/index.js',
    '/usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/validator/index.js',
    '/usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/mocker/callback/callbacks.js',
    '/usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/mocker/index.js',
    '/usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/forwarder/index.js',
    '/usr/local/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/dist/index.js',
    '/usr/local/lib/node_modules/@stoplight/prism-cli/dist/util/createServer.js',
    '/usr/local/lib/node_modules/@stoplight/prism-cli/dist/commands/mock.js',
    '/usr/local/lib/node_modules/@stoplight/prism-cli/dist/index.js'
  ]
}

```

## To Reproduce

1. Install the latest version by using
```
npm install -g @stoplight/prism-cli
```
2. Tried creating a mock server using a sample swagger specification.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -10,13 +10,18 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## Fixed
 
-- Prism is not returning an error anymore when trying to construct a schema for HTTP headers and query string with mixed cases property names [#1268](https://github.com/stoplightio/prism/pull/1268)
 - Prism will now refuse to start in case it will detect circular references. [#1270](https://github.com/stoplightio/prism/pull/1270)
 
 ## Changed
 
 - **BREAKING**: The `getHttpOperationsFromSpec` has been moved from the HTTP Package to the CLI package. If you're using Prism programmatically, this might require some code changes on your side. `getHttpOperationsFromResource` has been removed. [#1009](https://github.com/stoplightio/prism/pull/1009), [#1192](https://github.com/stoplightio/prism/pull/1192)
 - **BREAKING**: The `createClientFromOperations` is now exported as `export function` instead of exporting an object. If you're using Prism programmatically, this might require some code changes on your side [#1009](https://github.com/stoplightio/prism/pull/1009)
+
+# 3.3.6 (2020-07-08)
+
+## Fixed
+
+- Prism is not returning an error anymore when trying to construct a schema for HTTP headers and query string with mixed cases property names [#1268](https://github.com/stoplightio/prism/pull/1268)
 
 # 3.3.5 (2020-05-26)
 

